session.go

```diff
@@ -363,7 +363,7 @@ func (s *session) send(frame []byte) (open bool) {
 	snd := &sender{
 		session:        s,
 		coalescedBytes: 0,
-		coalesced:      1,
+		coalesced:      0,
```

Comment:
Changing this back and running `go test -run TestPadding -v` again should be able to see the difference.